devel/ci-run-e2e.sh

```diff
@@ -30,14 +30,29 @@ export_logs() {
 SCRIPT_ROOT="$( cd "$( dirname "${BASH_SOURCE[0]}" )" > /dev/null && pwd )"
 export REPO_ROOT="${SCRIPT_ROOT}/.."
 source "${SCRIPT_ROOT}/lib/lib.sh"
+IS_OPENSHIFT=${IS_OPENSHIFT:-"false"}
+OPENSHIFT_VERSION=${OPENSHIFT_VERSION:-"4"}
```

Comment:
3?

Reply:
I had hope to support 4 too back then, replacing it